pyperformance/_benchmark.py

```diff
@@ -164,6 +164,10 @@ def runscript(self):
     def extra_opts(self):
         return self._get_metadata_value('extra_opts', ())
 
+    @property
+    def setup_py(self):
+        return self._get_metadata_value('setup_py', False)
```

Comment:
Let's return the filename (and check if the file exists) here:

```suggestion
    @property
    def setup_py(self):
        try:
            return self._setup_py
        except AttributeError:
            self._setup_py = None
            if self._get_metadata_value('install_setup', True):
                filename = self._get_metadata_value('setupscript', 'setup.py')
                rootdir = os.path.dirname(self.metafile)
                #rootdir = self._get_rootdir()
                filename = os.path.join(rootdir, filename)
                if os.path.isdir(filename):
                    filename = os.path.join(filename, 'setup.py')
                if os.path.exists(filename):
                    self._setup_py = filename
            return self._setup_py
```

Also, I'm thinking it might be best for the default "install_setup" to be `True`.  So, by default, the setup script is installed if it is there.  That seems like the more likely user expectation.

Reply:
See above for why the filename may not be necessary.

But no harm in changing the default to `True` (I think).

Reply:
see https://github.com/python/pyperformance/pull/197#discussion_r930154904